service/src/integrationTest/kotlin/fi/espoo/evaka/koski/KoskiPayloadIntegrationTest.kt

```diff
@@ -846,4 +846,102 @@ class KoskiPayloadIntegrationTest : FullApplicationTest(resetDbBeforeEach = true
         """
         JSONAssert.assertEquals(expected, stored.payload, JSONCompareMode.STRICT)
     }
+    @Test
+    fun `simple preschool club placement in 2019`() {
```

Comment:
I think this test should be implemented as a normal integration test in KoskiIntegrationTest.kt, instead of a "raw payload" test. These raw tests are verbose and brittle, and they only exist as a sanity check to guarantee we don't break the JSON format accidentally.

Reply:
Test moved and simplified.